tsl/src/nodes/columnar_scan/columnar_scan.c

```diff
@@ -50,11 +51,13 @@ typedef struct ColumnarScanState
 {
 	CustomScanState css;
 	VectorQualState vqstate;
+	ExprState *segmentby_qual;
```

Comment:
Name is very similar to the list of quals below. You might want to use a better name to avoid confusion.